app/scripts/components/modeldetails.js

```diff
@@ -58,6 +58,21 @@ var exports = (function () {
 
           return niceStrings[this.activeModel.data.shared];
         }
+      },
+      delft3DVersion: {
+        cache: false,
+        get: function () {
+          return this.activeModel.data.versions.delft3d.delft3d_version;
+        }
+      },
+      reposUrl: {
+        cache: false,
+        get: function () {
+          if (this.activeModel.data.versions.preprocess.REPOS_URL !== undefined) {
```

Comment:
`_.has(this.activeModel, 'data.versions.preprocess.REPOS_URL')` is a bit more robust